guides/source/routing.md

```diff
@@ -644,6 +644,7 @@ You can also use this to override routing methods defined by resources, like thi
 
 ```ruby
 get ':username', to: 'users#show', as: :user
+resources :users
```

Comment:
Can you update the text above to mention that order matters?

Reply:
I'm not sure why the order matters and what triggers the override (is it the `to:` or the `as:`?) but this would make sense to me.